lib/address_geocoder.rb

```diff
@@ -133,9 +152,9 @@ def get_final_url(level_of_search)
   def get_format_levels
     levels  = [1,5]
```

Comment:
Maybe you might create a constant to represent to number.
e.g `AdminstrationLevel = {'province' = 7, 'state' = 5}`
instead of [1,5], it will be `[AdminstrationLevel['province], AdminstrationLevel['state']]`
more readable to me. :smile: